Sprint 7 solution in file manager

gennadiyniki opened the pull request.

gennadiyniki committed:
ФЗ7

gennadiyniki committed:
ФЗ7

gennadiyniki committed:
ФЗ7

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

mikita95 commented on src/manager/CSVFormatter.java:
```diff
@@ -0,0 +1,64 @@
+        builder.append(task.getName()).append(",");
+        builder.append(task.getTaskStatus()).append(",");
+        builder.append(task.getDescription()).append(",");
+        if (task instanceof Subtask subtask) {
```
В текущем задании не нужно использовать instanceof, нужно использовать enum типов задач

gennadiyniki replied:
> В текущем задании не нужно использовать instanceof, нужно использовать перечисление типов задач

привет!
про  instanceof понял, поправил, не понимаю пока со сборкой, так как в IDEA все тесты без ошибок проходят.

Небольшой апдейт!
Исправил, добавил путь к файлу 
Path path = Paths.get(getClass().getClassLoader().getResource("data.txt").toURI());

для сборки здесь помогло, в мой Managers добавить отдельный метод для тестов без файла, но честно говоря не особо понял принципа этого.

    public static TaskManager getInMemoryManager() {
        return new InMemoryTaskManager();
    }

mikita95 reviewed (changes requested):
Все хорошо, оставил небольшие замечания :)
Также сборка не проходит.

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

gennadiyniki committed:
ФЗ7

# Conflicts:
#	src/manager/Managers.java
#	test/manager/FileBackedTaskManagerTest.java
#	test/manager/InMemoryTaskManagerTest.java

mikita95 reviewed (approved):
Все отлично :) Можно мержить пул реквест
